stress/shell/src/main/java/alluxio/stress/cli/fuse/FuseIOBench.java

```diff
@@ -320,32 +375,38 @@ private void runInternal() throws Exception {
       mStartBarrierPassed = true;
 
       if (mParameters.mOperation == FuseIOOperation.LIST_FILE) {
+        for (String workerDirNames: mWorkerDirNames) {
+          for (int dirId = mThreadId; dirId < mParameters.mNumDirs; dirId += mParameters.mThreads) {
+            String dirPath = String.format("%s/%s/dir-%d", mParameters.mLocalPath,
+                workerDirNames, dirId);
+            File dir = new File(dirPath);
+            dir.listFiles();
+          }
+        }
+      } else if (mParameters.mOperation == FuseIOOperation.WRITE
+          || mParameters.mOperation == FuseIOOperation.LOCAL_READ) {
         for (int dirId = mThreadId; dirId < mParameters.mNumDirs; dirId += mParameters.mThreads) {
-          String dirPath = String.format("%s/%d", mParameters.mLocalPath, dirId);
-          File dir = new File(dirPath);
-          dir.listFiles();
+          for (int fileId = 0; fileId < mParameters.mNumFilesPerDir; fileId++) {
+            String filePath = String.format("%s/%s/dir-%d/file-%d", mParameters.mLocalPath,
```

Comment:
for small files, i think string concatenation could largely impact the throughput performance
any ways to improve

Reply:
To open the file we kind of have to have the path string. To construct this string I think this is the best we can do.

Reply:
Seems like we could use either "+" or StringBuilder, which are basically the same. https://stackoverflow.com/questions/513600/should-i-use-javas-string-format-if-performance-is-important

Reply:
Using StringBuilder will look like this:
```
String filePath = new StringBuilder(mParameters.mLocalPath)
        .append("/")
        .append(mBaseParameters.mId)
        .append("/dir-")
        .append(testDirId)
        .append("/file-")
        .append(testFileId)
        .toString();
```
Using "+" will look like this:
```
String filePath = mParameters.mLocalPath + "/" + mBaseParameters.mId + "/dir-"
        + testDirId + "/file-" + testFileId;
```
I think either is fine. What do you think? @LuQQiu

Reply:
I don't have pretty good suggestions. We can keep the current impl, just to remind us that if future we see some performance number that is not expected, it could be the string concatenation issue especially when we are testing small files

Reply:
It is surprising that Stirng.format() is >10 times slower than the other two.